Lib/turtle.py

```diff
@@ -140,7 +140,7 @@
 _tg_utilities = ['write_docstringdict', 'done']
 
 __all__ = (_tg_classes + _tg_screen_functions + _tg_turtle_functions +
-           _tg_utilities + ['Terminator']) # + _math_functions)
+           _tg_utilities) # + _math_functions)
```

Comment:
Perhaps TurtleGraphicsError, defined just after Terminator, should have and should be included instead of Terminator.

Reply:
Should it be this way?
```
__all__ = (_tg_classes + _tg_screen_functions + _tg_turtle_functions +
           _tg_utilities + ['Terminator'] +TurtleGraphicsError ) # + _math_functions)
```